_posts/2022-09-23-discrete-events-simulation-at-providence-outpatient-care-services-covid-19-recovery-scheduling/des-for-covid-19-recovery-scheduling.Rmd

```diff
@@ -0,0 +1,80 @@
+DES has long been proven successful in manufacturing, services, and supply chain sectors in solving complex resourcing and efficiency questions.  With growing concern on increasing costs to provide healthcare, and perceived lack of efficiency, there is growing usage of DES in healthcare.
+DES is commonly used to test schedules (for operating rooms, nursing, emergency department staffing, or other resourcing), test equipment purchase decisions (how many IV pumps are required to provide full coverage to the hospital?), space requirements (how many patient rooms, waiting chairs, do we need in this ED?) and even managing patient admissions and scheduling procedures.
+
+At Providence, with the second Covid-19 pandemic dwindling, we wanted to test schedules to cut down the growing backlog of outpatient appointments required to fill while ensuring safe social distancing practices within the hospital.
+
+
+## Background
+
+The Providence Healthcare Center(PHC) Outpatient Service Department consists of multiple clinics providing medical appointments.  Each patient arrives, registers, and wait for the appointment in the waiting areas or vehicle.  They will then be escorted to clinical waiting areas or directed into a room.  After the appointment, they may continue to wait in the clinical waiting areas for follow ups or leave the hospital.  During this process, there may be congestion resulting from a combinations of patient behaviors, system resource and capacity (early-arrival of wheel-trans patients, max room capacity, lack of therapists); A smooth outpatient process is important for the hospital system to run efficiently for optimal patient care and quality of service.
+
+< Insert flow diagram here >
+
+Under COVID regulations, all individuals are required to maintain social distancing between each other.  To follow the regulation and minimize unnecessary contacts, the Outpatient Service Department needs to control the maximum number of patients in each area of the department.  
+Planning to reopen more outpatient services (and to reduce the backlog created by shutdowns during the first two waves of the pandemic), the outpatient department wants to test adding additional appointments to their schedule with a combination of more staffing, longer hours, more patients and additional working days, while maintaining social distancing requirements.  
+
+## Solution
+
+The solution that was deployed for the PHC Outpatient Service Department consists of two parts:
+
+1) A simulation model that helps recreate the physical space capacity and analyze the system & process performance
+
+2) designing a web user interface that requires users to input key features of the simulation for testing different scenarios. In addition, the model would output key performance indicators to help access the performance of the proposed plans/system. 
```

Comment:
what are some of the input features that can be used to test different scenarios?

Reply:
I clarified that the primary input was the appointment schedule and added some detail of what this schedule contains.